sounds/management/commands/analysis_orchestrator.py

```diff
@@ -0,0 +1,96 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+import json
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from sounds.models import Sound, SoundAnalysis
+
+console_logger = logging.getLogger("console")
+
+# Dictionary that links simple names and versions of the analyzers to their actual docker images
+# Needs to be updated everytime a new analyzer or analyzer version is added
+ANALYZERS_DICT = {
+    "fs-essentia:1": "fs-essentia-extractor:20210525_9a3bd10",
+    "audio-commons:1": "ac-extractor:20210525_9a3bd10",
+    "audioset-vggish:1": "audioset-vggish-extractor:20210615_f44030e"
+}
+
+
+class Command(BaseCommand):
+
+    help = """Checks if there are sounds that have not been analyzed by the analyzers defined in the Analysis 
+    Configuration File (or that are analyzed with older versions) and send jobs to the analysis workers if needed"""
+
+    def add_arguments(self, parser):
```

Comment:
Another useful argument to have would be something like `--max-per-analyzer` which will allow to set a maximum number of analysis jobs to create per analyzer.